docs/Configuration.rst

```diff
@@ -2828,7 +2831,26 @@ To disable the activity type, simply set the parameter to ``false``.
 			   a content is shared in a space.  
 			-  **exosocial\\:relationship**: Activity post when two user are
 			   connected together.
+		
+.. _Configuration.ActivityLinkPreview:		
+		
 			    
+==========================================
+Enabling/Disabling activity link preview
+==========================================
+
+eXo Platform allows you to enable or disable the link preview in an activity 
+by defining the parameter ``exo.activity.link.preview.enabled`` 
+in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.
+
+.. code::
+
+     #Enable/Disable Activity link preview
+     exo.activity.link.preview.enabled=true
+     
+By default, the parameter is set to true i.e. the activities link preview
+is enabled.
```

Comment:
Shouldn't we add screenshots to make it clear what preview is, and what is the result when it is disabled ?

Reply:
ok, i will apply the patch to make screenshots